internal/pkg/docker/dockerengine/dockerengine.go

```diff
@@ -31,6 +31,8 @@ const (
 
 	ArchAMD64 = "amd64"
 	ArchX86   = "x86_64"
+	ArchARM = "arm"
+	ArchARM64 = "arm64"
```

Comment:
Oh looks like `gofmt` might not be running on save in your IDE 💭  The formatting seems to be off here. This might help:

https://stackoverflow.com/a/45590498/1201381

Reply:
🤦🏼